xml/suse_l10n.xml

```diff
@@ -93,9 +95,70 @@
   The individual components are connected by special characters:
  </para>
 
-<screen>
-  LANG=&lt;language&gt;[[_&lt;COUNTRY&gt;].&lt;Encoding&gt;[@&lt;Modifier&gt;]]
-</screen>
+<screen>LANG=&lt;language&gt;[[_&lt;COUNTRY&gt;].&lt;Encoding&gt;[@&lt;Modifier&gt;]]</screen>
+
+ <sect2 xml:id="sec.suse.l10n.settings">
+  <title>System Wide Locale Settings</title>
+  <para>
+   Since &sls; 15, &systemd; reads <filename>/etc/locale.conf</filename>
+   at early boot. The locale settings configured in this file are
+   inherited by every service or user, unless there are individual
+   settings.
+  </para>
+
+  <para>To set the system-wide locale, you could either:</para>
+
+  <itemizedlist>
+     <listitem>
+      <para>
+       Write your settings in <filename>/etc/locale.conf</filename>.
+       Each line is a environment-like variable assignment (see <xref
+        linkend="vl.suse.l10n.variables"/> for a list of variables).
+       For example, to set the language to German in Germany, use:
```

Comment:
Actually yes. If you look into some manpages, you will find `LANG=de_DE.UTF-8`. So you *have* the location there!

I think, your example with American English is not ideal (because the text is already in English). IMHO, in this case it does make more sense to use a different locale.

Reply:
Again, this is not the location (Germany) but the language (German). For example, Swiss German is different from German German (e.g. the Swiss they don't have the "ß").

You are free to use any language you want for the example(s), but please name it correctly. "German in Germany" is _not_ correct, it would be "German German", which sounds odd. That's why I proposed something else.

Reply:
> For example, Swiss German is different from German German (e.g. the Swiss they don't have the "ß").

Yes, I that's clear.

> You are free to use any language you want for the example(s), but please name it correctly. "German in Germany" is not correct, it would be "German German", which sounds odd. 

Maybe the confusion comes from the overloaded term "language". You see *language* as pure natural language, like English, German, etc. ("The language has nothing to do with the location!")

I see *language* more as a locale, how the `LANG` variable is defined. That definition clearly states: language plus country/territory (I skipped the rest for the sake of simplicity). You have a language (German) which is used in a specific country (Switzerland, Austria, Germany...).

Maybe we should better avoid language as it is so overloaded and confusing use *locale*. With that in mind, what about this:

```suggestion
       For example, to set the locale to German as used in Germany, write:
```

I think, this makes it clear what is is meant and it also adheres to the definition. We can be both happy! :wink: 

IMHO, your example with American English is an imprecise. *America* is a continent, but we colloquially use it as a synonym for USA.

Reply:
I think "as used in German" is just as bad as "German in Germany". German German is the correct term for the language, the only problem is that it sounds redundant. I don't care what example you choose, just use something that doesn't sound stupid/redundant.

Reply:
Well, I revamped the sentence and reduced it to "For example".